packages/jest-snapshot/src/State.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+'use strict';
+
+import type {Path} from 'types/Config';
+
+const {SnapshotFile, forFile} = require('./SnapshotFile');
```

Comment:
since SnapshotFile isn't used directly, it should be imported using `import type` instead.